### PR TITLE
add as_strided to PyTorch Frontends #10128

### PR DESCRIPTION
**Pull request by musthofasyarifudin**
add as_strided #10128

### CODE REVIEW

**Comment by musthofasyarifudin** [comment]
hi

**Comment by musthofasyarifudin** [comment]
hi can you help me. 
1. test metric tensorflow there is some lint error should i fixed it or leave as it is? (i dont change anything before)
2.  in test creation.py there is other fail test before i implement my own function do i need to fix those too

**Comment by musthofasyarifudin** [comment]
Hi @agniiyer can you review my PR please, let me know if i need something to fix.

Thanks

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by agniiyer** [comment]
Nicely done! :)
